.vscode/settings.json

```diff
@@ -1,6 +1,6 @@
 {
   "editor.fontLigatures": "'calt', 'liga', 'ss01', 'ss02', 'ss03', 'ss04', 'ss05', 'ss06', 'ss07', 'ss08', 'ss09'",
-  // "editor.formatOnSave": true,
+  "editor.formatOnSave": true,
```

Comment:
Let's leave commented out (or remove it since it's not used) since Solidity `forge fmt` and Go code are formatted differently.

Reply:
Removed in https://github.com/synapsecns/sanguine/pull/3085/commits/a49b9ad0af4a46c30fdc1d3085f49036c5125899